pkg/validate/validate.go

```diff
@@ -113,6 +120,11 @@ func BuildRunFields(buildRun *build.BuildRun) (string, string) {
 			return resources.BuildRunBuildFieldOverrideForbidden,
 				"cannot use 'timeout' override and 'buildSpec' simultaneously"
 		}
+
+		if buildRun.Spec.BuildSpec.Trigger != nil {
+			return resources.BuildRunBuildFieldOverrideForbidden,
+				"cannot use 'triggers' override in the 'BuildRun', it allowed in the 'Build'"
```

Comment:
```suggestion
				"cannot use 'triggers' override in the 'BuildRun', only allowed in the 'Build'"
```

wondering if this is more accurate?